src/plot_topoplotseries.jl

```diff
@@ -32,54 +35,88 @@ Multiple miniature topoplots in regular distances.
     Enable interactive mode. \\ 
     If you create `obs_tuple = Observable((0, 0, 0))` and pass it into `interactive_scatter` you can change observable indecies by clicking topopplot markers.\\
     `(0, 0, 0)` corresponds to the indecies of row of topoplot layout, column of topoplot layout and channell. 
+- `mapping.layout = nothing`\\
+    When equals `:time` arrange topoplots by rows. 
+
 
 $(_docstring(:topoplotseries))
 
 **Return Value:** `Figure` displaying the Topoplot series.
 """
-plot_topoplotseries(data::DataFrame, Δbin::Real; kwargs...) =
-    plot_topoplotseries!(Figure(), data, Δbin; kwargs...)
+plot_topoplotseries(data::DataFrame; kwargs...) =
+    plot_topoplotseries!(Figure(), data; kwargs...)
 
 function plot_topoplotseries!(
     f::Union{GridPosition,GridLayout,Figure,GridLayoutBase.GridSubposition},
-    data::Union{<:Observable{<:DataFrame},DataFrame},
-    Δbin;
+    data::Union{<:Observable{<:DataFrame},DataFrame};
+    bin_width = nothing,
+    bin_num = nothing,
     positions = nothing,
-    labels = nothing,
+    nrows = 1,
+    labels = nothing, # rename to channel_labels?
     combinefun = mean,
-    col_labels = true,
+    col_labels = false,
```

Comment:
Strictly speaking this is breaking, why did you do it? But I'm fine with it not breaking, it's a small change

Reply:
because by default user need to see time bins. col_labels = true would prevent that

Reply:
ok please note it down as a changing effect